filters/openpolicyagent/response.go

```diff
@@ -0,0 +1,76 @@
+package openpolicyagent
+
+import (
+	"bytes"
+	"io"
+
+	"net/http"
+
+	"github.com/open-policy-agent/opa-envoy-plugin/envoyauth"
+	"github.com/opentracing/opentracing-go"
+	"github.com/zalando/skipper/filters"
+)
+
+func (opa *OpenPolicyAgentInstance) RejectInvalidDecisionError(fc filters.FilterContext, span opentracing.Span, result *envoyauth.EvalResult, err error) {
```

Comment:
You log errors in the span, but the span is not marked as error via `span.SetTag("error", true)`, which is likely how people want to filter their streams to observe reject rates.

Reply:
Good catch! We still have to run Skipper with enabled tracing to capture how practical the traces are.